### PR TITLE
code structure reference updated with new changes

### PR DESCRIPTION
**Pull request by Madhura-saw**
### Description

In this PR I have modified the code structure reference in the README.md file. The following are the key changes made in the code structure:
1. Removed the `events` reference
2. Added the completed updated structure under the `tests/unit` path
3. Added the `docs` path and a description for it
4. Added `install.py`, `setup.py` and `requirements.txt`
5. There was a typo in the screenshot description which I corrected
6. I have also updated the `screenshot` reference
7. I have added the path for the `labs` and `presentation` reference
8. Replaced the `CONTRIBUTION.md` with `CHANGELOG.md`

- Relevant Issues : #127 
- Type of change :
  - [ ] New feature
  - [ ] Bug fix for existing feature
  - [ ] Code quality improvement
  - [ ] Addition or Improvement of tests
  - [x] Addition or Improvement of documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,61 +26,60 @@ Modelled as a state machine, with the help of AWS step functions, Trapheus resto
...
 ## Table of Contents
 
-- * [➤ Pre-Requisites](#pre-requisites )
-- * [➤ Parameters](#parameters)
-- * [➤ Instructions](#instructions)
-- * [➤ Execution](#execution)
-- * [➤ How it Works](#how-it-works)
-- * [➤ Contributing to Trapheus](#contributing-to-trapheus)
-- * [➤ Contributors](#contributors)
+- - [➤ Pre-Requisites](#pre-requisites)
```
Comment (namitad) [inline README.md:29]: @Madhura-saw did you make these changes? they seem to be unrelated to your PR.

Ideally i was expecting a diff only from this line https://github.com/intuit/Trapheus/pull/147/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R243

Can we please remove the unrelated changes to make this easier for review?

File: README.md [hunk 2]
```diff
@@ -239,20 +240,55 @@ Reference Code Structure
 
 ```bash
 
-├── CONTRIBUTING.md                               <-- How to contribute to Trapheus
+├── CHANGELOG.md
```
Comment (namitad) [inline README.md:243]: we can remove CHANGELOG.md

File: README.md [hunk 3]
```diff
@@ -239,20 +240,55 @@ Reference Code Structure
...
 ├── README.md                                     <-- The Readme file.
-├── events
-│   └── event.json                                <-- JSON event file to be used for local SAM testing
-├── screenshots                                   <-- Folder for screenshots of teh state machine.
-│   ├── Trapheus-logo.png
+├── docs                                          <-- The Readme files.
+│   ├── README.fr.md
+│   └── README.zh-CN.md
+├── install.py
+├── labs
```
Comment (namitad) [inline README.md:250]: lets remove the  substructure under`labs` - because it is still a work in progress

**Review by namitad — CHANGES_REQUESTED** [review]
remove the large diff from the file since they are not related to given issue

File: README.md (hunk 2 above)
Comment (Madhura-saw) [reply README.md:243]: sure

File: README.md (hunk 3 above)
Comment (Madhura-saw) [reply README.md:250]: ok

File: README.md (hunk 1 above)
Comment (Madhura-saw) [reply README.md:29]: No, I did not make any changes over there

**Comment by Madhura-saw** [comment]
> remove the large diff from the file since they are not related to given issue

I am sorry but I didn't understand what you meant here

**Comment by namitad** [comment]
@Madhura-saw if you see the changes/diff in the files, there are lot of changes unrelated your PR. Can you try taking the latest code and rebase so that the PR shows only your changes accurately?

**Comment by Madhura-saw** [comment]
> @Madhura-saw if you see the changes/diff in the files, there are lot of changes unrelated your PR. Can you try taking the latest code and rebase so that the PR shows only your changes accurately?

yes, on it

**Comment by Madhura-saw** [comment]
I have made a new commit, I think the previous changes were because when I saved the file in vscode it applied some formatting on it.

**Review by namitad — APPROVED** [review]
LGTM

**Comment by namitad** [comment]
@Madhura-saw  Thank you for your first contribution to Trapheus! I have added the required hacktoberfest labels as well.
Looking forward to more contributions!

**Comment by Madhura-saw** [comment]
> @Madhura-saw Thank you for your first contribution to Trapheus! I have added the required hacktoberfest labels as well. Looking forward to more contributions!

Thank you :)
